test/exposed/EverlongExposed.sol

```diff
@@ -0,0 +1,23 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+import { EverlongBase } from "../../contracts/internal/EverlongBase.sol";
+import { Everlong } from "../../contracts/Everlong.sol";
+import { console2 } from "forge-std/console2.sol";
+import { Test } from "forge-std/Test.sol";
+
+/// @title EverlongExposed
+/// @dev Exposes all internal functions for the `Everlong` contract.
+contract EverlongExposed is Everlong, Test {
```

Comment:
Left it as a stub for later when/if there's internal methods we want to expose.

I feel like your comment "We could just expose all of the internal functions here and avoid having all the different files." brings up something that I've been trying to battle with and don't really have a good solution for yet... 

I'm trying to isolate functionality to multiple smaller contracts since it would be easier to reason-about / explore as opposed to having one big gigaharness.... any suggestions on how we could accomplish this?

Was initially thinking that the various exposed contracts could be composed into the bigger `EverlongExposed` but as it stands that won't work.

Reply:
Gotcha, I figured that might be the case. I think we can accomplish this via inheritance. What I'd suggest is the following:

1. We keep separate exposed contracts for each of the "mixins."
2. We tie these together into a single `EverlongExposed` contract by adding a big `is` declaration to this contract.
3. When we write tests, we only use `EverlongExposed` and don't have separate contracts.

IMO, this strikes a good balance between searchability and good testing UX. When I write tests, I don't want to have to remember which exposed contract I want, and I definitely don't want to run into cases where I need functions from two different exposed contracts. That said, I totally agree that it's nicer to read through the harness if it's logically split up. The only reason I don't have a problem with this in Hyperdrive is that I wrote it and know where everything is already.

Reply:
Ofc, let me know what you think. If you disagree, I'd like to hear it.

Reply:
I like this a lot! Will do